### ISSUE TITLE
Business - Wollboutique Reinheim

### ISSUE THREAD

**Issue by corebean** [zeroesapps/ladenfreude#19]
Wollboutique Reinheim
Kirchstrasse 10, 64354 Reinheim
06162 9166192
wollboutique2014@online.de

Opening hours to come, but please create with these details for now.

**Comment by corebean** [comment]
Can you please add the opening times too?

Mo, Tu, Th Fr = 10:00-13:00 and again from 15:00-18:00. Saturday from 10:00-13:00.

**Comment by corebean** [comment]
Oh and I know the username will always be the email address, but are we setting a standard password for all accounts, or generating secure ones, or letting them set their own?

**Comment by russog** [comment]
atm we have no way to set it from the app. I can start making stronger passwords if you like....

**Comment by corebean** [comment]
No stronger passwords are not necessary for now, because people need to
remember them and we are just in Alpha anyway. Just set the password to
something easy and I can hand it out as the accounts are setup.

I'll mark it however as a future feature.

**Comment by corebean** [comment]
So what is the username/password for this account? I tried the email address and used "password123" but it says it's not accepted.

**Comment by corebean** [comment]
Reopening because there are a few things wrong.
    •    SHOP is closed on Wednesday
    •    She doesn't have a website, so either the link should be deactivated, or we can supply the Facebook page. What makes more sense?

**Comment by russog** [comment]
ok removed the wed (check that it is fine now).
the link is your call. if we want to disable it you need to ask @bereczandor to put some logic to disable it when the url is null otherwise I can put the FB page....

**Comment by russog** [comment]
just added the url could you please check on close?

**Comment by corebean** [comment]
Nope - doesn't work - I get a weird message in my browser. Something about "unsafe..."

**Comment by russog** [comment]
@bereczandor could you please have a look why this would happen?

**Comment by corebean** [comment]
And please notify me of course when it's working.

**Comment by corebean** [comment]
Still getting this URL issue when clicking on the Website link which displays the message

"The address wasn’t understood"

I think the link contains a text like "unsafe:"

**Comment by bereczandor** [comment]
Yes i noticed that too, but i don't know yet why is The unsafe word prepended to The link. Normally a lunk is a link, should be accepted as it is. Perhaps because it's facebook?😁
